l3kernel/CHANGELOG.md

```diff
@@ -7,9 +7,19 @@ this project uses date-based 'snapshot' version identifiers.
 
 ## [Unreleased]
 
+### Added
+- `\box_set_clipped:N` and variants
+
+### Changed
+- `\box_set_trim:Nnnnn` and `\box_set_viewport:Nnnnn` moved to
+  stable kernel
+
 ### Fixed
 - Incorrect internal name in `\pdf_pagesize_gset:nn`
 
+### Removed
+- Experimental `\box_clip:N` and variants
+
```

Comment:
To better reflect it's a renaming, I personally perfer
```suggestion
### Changed
- Experimental `\box_clip:N` and variants renamed to `\box_set_clipped:N` and
  variants
- `\box_set_clipped:N`, `\box_set_trim:Nnnnn`, and `\box_set_viewport:Nnnnn`
  moved to stable kernel

### Fixed
- Incorrect internal name in `\pdf_pagesize_gset:nn`

```

Reply:
I see what you mean, but the idea in  a formal change log is that *anything* that 'appears' goes down in the `Added` and *anything* that 'disappears' goes down in `Removed`: the fact it's a rename is a detail from that point of view.

Reply:
I've clarified the renaming in the changelog